src/Simulator/IncrementalRedstoneSimulator/RedstoneWireHandler.h

```diff
@@ -269,7 +269,7 @@ namespace RedstoneWireHandler
 		a_Chunk.SetMeta(a_Position, Power);
 
 		// Notify all positions, sans YP, to update:
-		for (const auto & Offset : RelativeAdjacents)
+		for (const auto & Offset : RelativeLaterals)
```

Comment:
TODO: use this and add a separate UpdateAdjustedRelative for YM, the loop + if construct is not optimal.